### PR TITLE
Documentation fixups

### PR DESCRIPTION
**Pull request by themuflon**

### CODE REVIEW

File: docs/examples/ClasspathExample/README.md [hunk 1]
```diff
@@ -12,6 +12,3 @@ How to use the loom.config classpath array to specify source folders to compile.
 
 ## Screenshot
 ![ClasspathExample Screenshot](images/screenshot.png)
-
-## Code
-@insert_source
```
Comment (bengarney) [inline docs/examples/ClasspathExample/README.md]: Why is this removed?

**Comment by bengarney** [comment]
One nit, this needs a rebase.

File: docs/examples/ClasspathExample/README.md (hunk 1 above)
Comment (themuflon) [inline docs/examples/ClasspathExample/README.md]: Source is not in the standard directories, doc gen doesn't pick it up.

Comment (bengarney) [inline docs/examples/ClasspathExample/README.md]: Oh. Should we move the source or add an explanatory note in the ## Code section instead of deleting it entirely?

Comment (themuflon) [inline docs/examples/ClasspathExample/README.md]: Moving the source would be pointless because that's the point :) I'll copy-pase the relevant config contents, code is irrelevant here anyway.
